patterns/2-structured/core-team.md

```diff
@@ -0,0 +1,107 @@
+## Title
+
+Core Team
```

Comment:
When toying with names for this team and pattern, did you have any alternative names in mind?
I am thinking "Contribution Enablement Team" or so.

Especially the following sentence seems essential for the role that this team plays:

> The core team doesn't have its own business agenda that determines its contributions.